site/content/3.12/aql/fundamentals/subqueries.md

```diff
@@ -156,3 +219,36 @@ The additional fallback `maybe || {}` prevents a query warning
 
 that originates from a `null` value getting passed to the `ATTRIBUTES()`
 function that expects an object.
+
+Similarly, when you use subqueries as sub-expressions that are combined with
+logical `AND` or `OR`, the subqueries are always executed:
+
+```aql
+RETURN false AND (RETURN ASSERT(false, "executed"))
+```
+
+```aql
+RETURN true OR (RETURN ASSERT(false, "executed"))
+```
+
+If the first operand of a logical `AND` is `false`, the overall result is
+`false` regardless of the second operand. If the first operand of a logical `OR`
+is `true`, the overall result is `true` regardless of the second operand.
+However, the subqueries are run nonetheless, causing both example queries to fail.
+
```

Comment:
I don't think this is true anymore as of 3.12.1 & higher.

Reply:
It's in a tab with the title "Up to v3.10.0", see: https://deploy-preview-530--docs-hugo.netlify.app/stable/aql/fundamentals/subqueries/#evaluation-of-subqueries